test/index.js

```diff
@@ -1124,6 +1124,47 @@ describe('expect()', () => {
                 Hoek.assert(!exception, exception);
                 done();
             });
+
+            it('asserts called with only one argument', (done) => {
+
+                {
+                    let exception = false;
+                    try {
+                        Code.expect('abc').to.include();
+                    }
+                    catch (err) {
+                        exception = err;
+                    }
+                    Hoek.assert(exception.message === 'Can only assert include with a single parameter', exception);
+                }
+
+                {
```

Comment:
I think this second test can be dropped, as the functionality is tested elsewhere.